eventmesh-connectors/eventmesh-connector-dingtalk/README.md

```diff
@@ -0,0 +1,26 @@
+# eventmesh-connector-dingtalk
+
+## DingtalkSinkConnector：from eventmesh to dingtalk。
+
```

Comment:
It will be better ro remove this CN period and use `EventMesh` and `DingTalk` instead.

Reply:
Fixed in https://github.com/apache/eventmesh/pull/4668.